src/adagrams.js

```diff
@@ -1,15 +1,127 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
+/* eslint-disable quotes */
+/* eslint-disable no-unused-vars */
+
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
 };
 
+// Implement this method for wave 1
+export const drawLetters = () => {
+  const copyOfLetters = JSON.parse(JSON.stringify(LETTER_POOL))
+  let tenLetters = [];
+  while (tenLetters.length < 10) {
+    const keys = Object.keys(copyOfLetters)
+    let randomLetter = keys[Math.floor(Math.random() * keys.length)];
+    
+    if (copyOfLetters[randomLetter] > 0) {
+      tenLetters.push(randomLetter); 
+      copyOfLetters[randomLetter] -= 1;
+    }
+  }
+  return tenLetters;
+}
+
+
+// Implement this method for wave 2
 export const usesAvailableLetters = (input, lettersInHand) => {
-  // Implement this method for wave 2
+  let copyLettersInHand = lettersInHand;
```

Comment:
This doesn't create a copy but rather just points another variable at `lettersInHand`, so any mutations made to one also reflect in the other. Nothing breaks as the result of that here, but it probably isn't what you intend.